Libraries/LibWeb/Painting/ShadowPainting.cpp

```diff
@@ -94,11 +95,13 @@ void paint_text_shadow(PaintContext& context, PaintableFragment const& fragment,
             text_rect.height() + margin + margin
         };
 
+        // FIXME: this is close but not quite perfect. non integer scale values can be offset by tiny amounts.
+        auto device_margin = layer.blur_radius * 2;
```

Comment:
why multiply by 2? if you want to covert to device pixels then `device_pixels_per_css_pixel()` should be used.

Reply:
that is correct, no idea why i named it that :)